docs/release-notes/release-notes-0.15.0.md

```diff
@@ -16,6 +16,10 @@
   for running lnd alongside a bitcoind service is now provided in
   `contrib/init/lnd.service`.
 
+## Mobile
+
+* Improved instructions on [how to build lnd for mobile](https://github.com/lightningnetwork/lnd/tree/master/mobile).
```

Comment:
The release notes should also include a link to the PR itself, for easy reference of the actual change.